static/src/stylesheets/module/identity/_switchboard.scss

```diff
@@ -0,0 +1,212 @@
+        padding: $local-gutter * 1.5;
+        padding-left: $local-gutter * 2 + $checkbox-size;
+        padding-top: $local-gutter * .9;
+        position: relative;
+    }
+
+    &:hover {
+        background-color: darken($neutral-8, 1%);
+    }
+
+    @supports (background-size: cover) {
+        input[type=checkbox] {
+            visibility: hidden;
+            position: absolute;
+            top: 0;
+            right: 0;
+            & + .manage-account__switchboardCheckbox {
+                display: block;
+                width: $checkbox-size;
+                height: $checkbox-size;
+                border: 1px solid $neutral-3;
+                border-radius: 1.5px;
+                background-size: $checkbox-size / 1.5;
+                background: #ffffff no-repeat center center;
+                position: absolute;
+                top: $local-gutter;
+                left: $local-gutter;
+                cursor: pointer;
+            }
+            &:checked + .manage-account__switchboardCheckbox {
+                background-image: url('data:image/svg+xml;charset=utf-8,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%2210.79%22%20height%3D%228.61%22%20viewBox%3D%220%200%2010.79%208.608%22%3E%3Cpath%20fill%3D%22%23005689%22%20d%3D%22M3%206.58L10.23%200l.55.53L3%208.6h-.27L0%204.8l.55-.56%202.44%202.33z%22%2F%3E%3C%2Fsvg%3E')  }
+        }
+    }
+
+    .manage-account__switchboardTitle {
+        @include fs-bodyCopy(1);
+        font-weight: 400;
+        margin-bottom: $local-gutter * .5;
+        cursor: pointer;
+        display: block;
+    }
+
+    .manage-account__switchboardCopy {
+        cursor: pointer;
+        opacity: .9;
+    }
+
+    .manage-account__switchboardTidbit {
+        @include fs-textSans(1);
+        opacity: .75;
+        svg {
+            vertical-align: -10%;
+            margin-right: .25em;
+        }
+    }
+
+    .manage-account__switchboardFooter {
+        margin-top: $local-gutter * 2;
+        display: flex;
+        justify-content: space-between;
+        width: 100%;
+    }
+
+    &:before {
+        content: '\0020';
+        position: absolute;
+        display: block;
+        top: 0;
+        left: 0;
+        right: 0;
+        background-color: $guardian-brand;
+        height: 2px;
+        transform-origin: center;
+        opacity: 0;
+        transition: .2s;
+    }
+
+    &--branded {
+        display: flex;
+        flex-direction: row;
+        > * {
+            flex: 1 1 0;
+            width: auto!important;
+        }
+        .account__switchboardLabelHeader {
+            background: #f5b3c7;
+            color: #e71c77;
+        }
+    }
+
+    &.is-updating {
+        cursor: wait;
+        pointer-events: none;
+        background-image: none;
+
+        &:before {
+            animation: manage-account__switchboardLoading 1s infinite both;
+            opacity: 1;
+        }
+
+        > * {
+            opacity: .5;
+        }
```

Comment:
If we applied `opacity` to the `.is-updating` element, would the children inherit the opacity? It's possible that this will mess up the styles of the `.is-updating` element, in which case ignore this suggestion 😅

Reply:
sadly in this case it does mess them up :(